google/utils.go

```diff
@@ -229,6 +229,23 @@ func linkDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return false
 }
 
+// expandLabels pulls the value of "labels" out of a schema.ResourceData as a map[string]string.
+func expandLabels(d *schema.ResourceData) map[string]string {
```

Comment:
we use the naming convention `expand*` when converting from []interface{} or the ResourceData to the go client object. 

For example:
`func expandAliasIpRanges(ranges []interface{}) []*computeBeta.AliasIpRange`

I find it a bit confusing to reuse the same term for something different.